src/pages/index.js

```diff
@@ -156,23 +157,22 @@ function handleAvatarChange(data) {
       userInfo.setUserInfo(data)
     })
     .then(avatarPopup.close)
-    .finally(avatarPopup.renderLoading(false))
+    .finally(() => avatarPopup.renderLoading(false))
     .catch((err) => {
       console.log(console.error);
```

Comment:
`console.log(console.error);` should be `console.error(err);`. Please, pay attantion the `console.error` is a method for logging errors